### PR TITLE
cis: make service non-beta

### PR DESCRIPTION
**Pull request by lucasmoura**
## Proposed Commit Message
cis: make service non-beta

Turn CIS into a non-beta service and update the integration test to run the CIS script on the machine

## Test Steps
Run the modified integration tests

## Desired commit type::

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist:

 - [x] I have updated or added any unit tests accordingly
 - [x] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

**Review by orndorffgrant — COMMENTED** [review]
Just did a quick lookover and it looks great! I'll look again more in depth tomorrow

File: features/attached_enable.feature [hunk 1]
```diff
@@ -150,6 +150,52 @@ Feature: Enable command behaviour when attached to an UA subscription
...
+        \s* 500 https://esm.ubuntu.com/cis/ubuntu <release>/main amd64 Packages
+        """
+        When I verify that running `ua enable cis` `with sudo` exits `1`
+        Then stdout matches regexp
+        """
+        One moment, checking your subscription first
+        CIS Audit is already enabled.
+        See: sudo ua status
+        """
+        And I verify that running `/usr/share/ubuntu-scap-security-guides/cis-hardening/<cis_script> lvl1_workstation` `with sudo` exits `0`
```
Comment (blackboxsw) [inline features/attached_enable.feature:191]: Lets add a check before hardening:
  And I run `cis-audit level1_server` with sudo
  Then stdout matches regexp
   """
   Title   Ensure root PATH Integrity
   Rule    xccdf_com.ubuntu.xenial.cis_rule_CIS-6.2.6
   Result  pass
   """
   And stdout matches regexp:
    """
   Title   Ensure password expiration is 90 days or less
   Rule    xccdf_com.ubuntu.xenial.cis_rule_CIS-5.4.1.1
   Result  fail
   """

 Might be worth adding also a stdout check re-running cis-audit after your cis-hardening script to ensure that the one "failed" case you audited for was fixed by the harden script

**Review by blackboxsw — CHANGES_REQUESTED** [review]
Let's also add a test for use of cis-audit too. and maybe a post harden retest on cis-audit to see that it fixed a failed case.

**Comment by lucasmoura** [comment]
@blackboxsw Great call here. I have updated the code to add the `cis-audit` checks

**Review by blackboxsw — APPROVED** [review]
LGTM!

**Pull request merged**